scarab/service_test.go

```diff
@@ -0,0 +1,189 @@
+package scarab
+
+import (
+	"testing"
+	"time"
+
+	"github.com/dedis/cothority"
+	"github.com/dedis/cothority/omniledger/darc"
+	ol "github.com/dedis/cothority/omniledger/service"
```

Comment:
we should use `ol` in eventlog too, saves so many characters..

Reply:
I even thought we could have the omniledger-service being `package ol`...